src/Panel/Controller/Search.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Kirby\Panel\Controller;
+
+use Kirby\Cms\App;
+use Kirby\Toolkit\Escape;
+
+/**
+ * The Search controller takes care of the logic
+ * for delivering Panel search results
+ *
+ * @package   Kirby Panel
+ * @author    Nico Hoffmann <nico@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ * @internal
+ */
+class Search
+{
+	public static function files(
+		string $query,
```

Comment:
Actually  `$query` argument could be nullable but not for this method. This is valid issue for all methods for this class.